src/api.rs

```diff
@@ -13,8 +13,8 @@ use futures::stream;
 use futures::StreamExt;
 use futures::TryStreamExt;
 use once_cell::sync::Lazy;
-use prometheus::IntCounterVec;
```

Comment:
shoot sorry about the whitespace/formatting changes, must have run vscode's formatter, let me know if you want me to undo

Reply:
I have a `.rustfmt.toml` in the repo and apparently didn't `cargo fmt` in one of the more recent changesets - this is fine, it follows the config